app/master/slave_allocator.py

```diff
@@ -0,0 +1,57 @@
+from app.util.log import get_logger
+from app.util.ordered_set_queue import OrderedSetQueue
+from app.util.safe_thread import SafeThread
+
+
+class SlaveAllocator(object):
+    """
+    The SlaveAllocator class is responsible for allocating slaves to prepared builds.
+    """
+
+    def __init__(self, build_request_handler):
+        """
+        :type build_request_handler: BuildRequestHandler
+        """
+        self._logger = get_logger(__name__)
```

Comment:
Good point, I'll rename them now that the context is already described by the enclosing class.